eng/PatchConfig.props

```diff
@@ -139,6 +139,7 @@ Later on, this will be checked using this condition:
       Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv;
       Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets;
       Microsoft.AspNetCore.SignalR.Redis;
+      Microsoft.AspNetCore.HttpOverrides;
```

Comment:
I assume this should be in a new section for 2.1.41?

Reply:
@wtgodbe is going to handle that.